app/utils/bundle-importer.js

```diff
@@ -227,6 +227,11 @@ YUI.add('bundle-importer', function(Y) {
     _executeDryRun: function(records) {
       if (this._dryRunIndex === records.length - 1) {
         // Done executing the recordSet.
+        this.db.notifications.add({
```

Comment:
Driveby - it's nice to have a notification to tell you when it's done importing as large bundles can take a while.